.github/workflows/unittests.yml

```diff
@@ -48,9 +48,11 @@ jobs:
       - run: mamba list
       - run: conda config --show-sources
       - run: conda config --show
+      - run: python -m pip install --no-deps -e .
```

Comment:
**This comment is for documentation & to gather my thoughts. It is not a blocker.**

* This works the same way it does in [`tobler`](https://github.com/pysal/tobler/blob/080fca17773988d8f6104d96ffad61b6b436daad/.github/workflows/unittests.yml#L47), etc. and it still nags me that I can't figure out why it works in `spaghetti`, etc. but not here...
* maybe there are differences in `setup.py`...?
* see also actions/checkout#410 and this on [SO](https://stackoverflow.com/questions/65014768/cant-find-my-python-module-after-installing-on-github-actions)